WordPress/Classes/ViewRelated/Views/WPRichText/WPRichTextFormatter.swift

```diff
@@ -491,13 +491,21 @@ class AttachmentTagProcessor: HtmlTagProcessor {
     func attachmentForHtml(_ html: String, identifier: String) -> WPTextAttachment? {
 
         let attrs = attributesFromTag(html)
-
         let source = getSource(from: attrs)
+        let cssClass = getCssClass(from: attrs)
+
         let textAttachment = WPTextAttachment(tagName: tagName, identifier: identifier, src: source)
         textAttachment.attributes = attrs
         textAttachment.html = html
 
-        let size = getSize(from: attrs)
+        var size = getSize(from: attrs)
+
+        // `twemoji` without an assigned size become outsized images
+        // https://github.com/wordpress-mobile/WordPress-iOS/issues/15341
+        if (size.height == 0 || size.height == 0) && cssClass.contains("emoji") {
+                size = CGSize(width: 20, height: 20)
```

Comment:
```suggestion
            size = CGSize(width: 20, height: 20)
```

Reply:
GH is lagging out and won't let me take the commit suggestion. Making the change manually and pushing a commit.